tests/unit/featurizers/test_cgr.py

```diff
@@ -105,6 +107,9 @@ def rxn_smi(request):
         BondExpectation((0, 2), bond_reac_none=True, bond_prod_none=True),
         BondExpectation((1, 2), bond_reac_none=False, bond_prod_none=False),
     ],
+    "[CH4:1]>>[CH3:1].[H:2]": [
+        BondExpectation((0, 0), bond_reac_none=True, bond_prod_none=True),
```

Comment:
this felt a little hacky to me because it's not really a bond. And I don't know if this is expected to remain the default behavior forever.

Reply:
This is the way RDKit is implemented. I think it is fine. Especially, our code would never use u and v as the same number. Will this test reach the code you changed in reaction.py? If not, I suggest we remove this test here.

Reply:
I put this here because of the way the test suite is set up: right now, anything in `rxn_smis` is expected to also have an example in `reac_prod_maps` and in `bond_expect_imbalanced`. If we remove this test, we will have to add some exceptions which might make the testing suite unnecessarily more complicated. But, I agree this doesn't really test anything meaningful. Let me know what you think.

Reply:
Without making much change, it seems this is the only way we can do this test. Could you add a comment on this line to mention this is only for test purposes for graphs with zero edges?

Reply:
done!